media-gfx/openvdb/files/openvdb-11.0.0-NANOVDB_USE_OPENVDB.patch

```diff
@@ -0,0 +1,10 @@
+--- openvdb-11.0.0.orig/cmake/FindOpenVDB.cmake	2023-11-01 21:31:11.000000000 +0100
```

Comment:
Add comment explaining why this patch is needed in the patch file

Reply:
This hunk forcefully propagates a mandatory setting from bulding nanovdb with openvdb.

Reply:
please add these comments to the top of the patch